lib/typescript/components/general/Emoji/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Emoji = props => (
```

Comment:
@airyhq/codeowners-frontend New accessible `Emoji` component. Please use this one if you want to use emojis moving forward.